pymobiledevice3/services/dvt/instruments/core_profile_session_tap.py

```diff
@@ -566,6 +565,7 @@ def read(self, size: int) -> bytes:
 
 
 class CoreProfileSessionTap(Tap):
+    """"""
```

Comment:
Remove

Reply:
this is workaround for sphinx so the subsequent doctoring wont be included